Utils/azuretools-core/src/com/microsoft/azuretools/utils/WebAppUtils.java

```diff
@@ -399,23 +399,23 @@ public static String getJDKVersion(@NotNull final WebApp webApp) {
      * And we will return N/A for those kind of web apps.
      */
     public static String getWebContainer(@NotNull final WebApp webApp) {
-        final OperatingSystem operatingSystem = webApp.operatingSystem();
-        if (operatingSystem == OperatingSystem.WINDOWS) {
-            return String.join(" ", webApp.javaContainer(), webApp.javaContainerVersion());
-        }
-        if (operatingSystem == OperatingSystem.LINUX) {
-            final String linuxVersion = webApp.linuxFxVersion();
-            final String[] versions = linuxVersion.split("-");
-            if (versions.length != 2) {
+        switch(webApp.operatingSystem()) {
+            case WINDOWS:
+                String.join(" ", webApp.javaContainer(), webApp.javaContainerVersion());
+            case LINUX:
+                final String linuxVersion = webApp.linuxFxVersion();
+                final String[] versions = linuxVersion.split("-");
+                if (versions.length != 2) {
+                    return linuxVersion;
+                }
+                if (StringUtils.containsIgnoreCase(versions[0], "tomcat")) {
+                    return versions[0].replace("|", " ");
+                } else {
+                    return "N/A";
```

Comment:
Also use `DEFAULT_VALUE_WHEN_VERSION_INVALID` here?

Reply:
Use "N/A" to differentiate with invalid version. Anyway, we will just show the runtime info in next changes.